ocaml-lsp-server/src/hover_req.ml

```diff
@@ -27,15 +32,40 @@ let format_contents ~syntax ~markdown ~typ ~doc =
       in
       { MarkupContent.value; kind = MarkupKind.PlainText })
 
-let handle server { HoverParams.textDocument = { uri }; position; _ } =
+let handle server { HoverParams.textDocument = { uri }; position; _ } mode =
   Fiber.of_thunk (fun () ->
       let state : State.t = Server.state server in
       let doc =
         let store = state.store in
         Document_store.get store uri
       in
+      let verbosity =
+        let mode =
+          match (mode, Sys.getenv_opt "OCAMLLSP_HOVER_IS_EXTENDED") with
```

Comment:
I used this to quickly experiment from my editor. I know this isn't encouraged, but this is convenient to have at least while in development

Reply:
Sounds fine to me.